xml/ay_bigfile.xml

```diff
@@ -4214,16 +4152,149 @@ openssl x509 -noout -fingerprint -sha256
+        &lt;size&gt;max&lt;/size&gt;
+      &lt;/partition&gt;
+    &lt;/partitions&gt;
+    &lt;use&gt;all&lt;/use&gt;
+  &lt;/drive&gt;
+  &lt;drive&gt;
+    &lt;device&gt;/dev/sdb&lt;/device&gt;
+    &lt;disklabel&gt;none&lt;/disklabel&gt;
+    &lt;partitions config:type="list"&gt;
+      &lt;partition&gt;
+        &lt;raid_name&gt;/dev/md/0&lt;/raid_name&gt;
+      &lt;/partition&gt;
+    &lt;/partitions&gt;
+    &lt;use&gt;all&lt;/use&gt;
+  &lt;/drive&gt;
+  &lt;drive&gt;
+    &lt;device&gt;/dev/md/0&lt;/device&gt;
+    &lt;partitions config:type="list"&gt;
+      &lt;partition&gt;
+        &lt;mount&gt;/home&lt;/mount&gt;
+        &lt;size&gt;40G&lt;/size&gt;
+      &lt;/partition&gt;
+      &lt;partition&gt;
+        &lt;mount&gt;/srv&lt;/mount&gt;
+        &lt;size&gt;10G&lt;/size&gt;
+      &lt;/partition&gt;
+    &lt;/partitions&gt;
+    &lt;raid_options&gt;
+      &lt;chunk_size&gt;4&lt;/chunk_size&gt;
+      &lt;parity_algorithm&gt;left_asymmetric&lt;/parity_algorithm&gt;
+      &lt;raid_type&gt;raid1&lt;/raid_type&gt;
+    &lt;/raid_options&gt;
+    &lt;use&gt;all&lt;/use&gt;
+  &lt;/drive&gt;
+&lt;/partitioning&gt;</screen>
+    </example>
+
+    <para>
+     If you do not want to create partitions in the software RAID, just set
+     the <literal>disklabel</literal> to <literal>none</literal> as you would
+     do for a regular disk. In the example below, only the RAID
+     <literal>drive</literal> section is shown for simplicity's sake:
+    </para>
+
+    <example>
+     <title>RAID1 Without Partitions</title>
+<screen>&lt;drive&gt;
+  &lt;device&gt;/dev/md/0&lt;/device&gt;
+  &lt;disklabel&gt;none&lt;/disklabel&gt;
+  &lt;partitions config:type="list"&gt;
+    &lt;partition&gt;
+      &lt;mount&gt;/home&lt;/mount&gt;
+      &lt;size&gt;40G&lt;/size&gt;
+    &lt;/partition&gt;
+  &lt;/partitions&gt;
+  &lt;raid_options&gt;
+    &lt;chunk_size&gt;4&lt;/chunk_size&gt;
+    &lt;parity_algorithm&gt;left_asymmetric&lt;/parity_algorithm&gt;
+    &lt;raid_type&gt;raid1&lt;/raid_type&gt;
+  &lt;/raid_options&gt;
+  &lt;use&gt;all&lt;/use&gt;
+&lt;/drive&gt;</screen>
+    </example>
+
+   <sect3 xml:id="ay.partition_sw_raid_deprecated">
+    <title>Using the Deprecated Syntax</title>
+    <para>
+     Starting on &productname; 15 SP1 (or 15 SP0 if the installer self-update
+     feature is enabled), it is possible to partition a software RAID.
+     However, that scenario was not supported in previous versions and hence
+     the way to define a software RAID was slightly different.
+    </para>
+
+    <para>
+     This section defines how the old-style configuration looks like because
+     it is still supported for backward compatibility.
+    </para>
+
+    <para>
+     Keep the following in mind when configuring a RAID using this deprecated
+     syntax:
+    </para>
+    <itemizedlist mark="bullet" spacing="normal">
+     <listitem>
+      <para>
+       The device for raid is always <literal>/dev/md</literal>
```

Comment:
The device for raid is always <literal>/dev/md</literal>. (end with full stop because the following list items use the same structure)